content/en/docs/instrumentation/swift/automatic.md

```diff
@@ -0,0 +1,126 @@
+---
```

Comment:
Just a reminder that the file needs to be renamed. I think that `libraries.md` will do, right @cartermp et al.?